source/features/parse-backticks.tsx

```diff
@@ -23,6 +23,8 @@ const selectors = [
 	'.js-hovercard-content > .Popover-message .Link--primary', // Hovercard
 	'.js-discussions-title-container h1 > .js-issue-title', // `isDiscussion`
 	'a[data-hovercard-type="discussion"]', // `isDiscussionList`
+	'.search-title', // `isGlobalSearchResults` search titles
```

Comment:
This element includes SVG too, we should not pass unexpected HTML to the linkifier: https://github.com/search?q=repo%3Arefined-github%2Frefined-github+is%3Aissue&type=issues

<img width="382" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/4313e475-e130-4d42-b1f4-32309bd1d522">




on the discussions there's no icon, which is fine: https://github.com/search?q=repo%3Acommunity%2Fcommunity+backticks+in+titles&type=discussions

<img width="393" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/71ad71af-dedb-4c85-af4b-67e6bef71ef9">


I think this can be easily fixed:

```suggestion
	'.search-title a', // `isGlobalSearchResults` search titles
```